.env.example

```diff
@@ -1 +1,2 @@
 SECRET_KEY='dwellingly'
+PYTHONPATH=${PWD}:${PYTHONPATH}
```

Comment:
I don't see this being used anywhere. What is this for?

Reply:
I was having trouble importing 'app' and 'db' as a modules in the alembic "env.py" file. The solution was to include a python path as one of the environmental variables. Apparently this is a common issue. The [recommended solution](https://stackoverflow.com/questions/15648284/alembic-alembic-revision-says-import-error) is to update the PYTHONPATH in the environmental variables.

Reply:
Thanks, I will check it out.